data_pipelines_cli/cli_commands/publish.py

```diff
@@ -0,0 +1,124 @@
+import glob
+import pathlib
+from typing import Any, Dict, List, Tuple
+
+import click
+import yaml
+
+from ..cli_constants import BUILD_DIR, PROFILE_NAME_ENV_EXECUTION
+from ..cli_utils import echo_subinfo
+from ..config_generation import get_profiles_yml_build_path
+from ..data_structures import DbtModel, DbtSource
+from ..dbt_utils import read_dbt_vars_from_configs
+from ..errors import DataPipelinesError
+from .prepare_env import replace_profiles_vars_with_values
+
+DBT_DATABASE_KEY_BY_PROVIDER = {
+    "bigquery": "project",
+    "postgres": "dbname",
+    "redshift": "dbname",
+    "snowflake": "database",
+}
```

Comment:
We should find a better way for this. What if someone wants to add mysql or spark? He will have to modify the code. DBT have his own abstraction as database, maybe we can extract it somehow using dbt lib?

Reply:
I think I found a way. Ws can (and probably should) use manifest.json instead of config files and yml files. There is a full list of models with schema, database, desc etc.